Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Beckn/OnInit.hs

```diff
@@ -72,6 +75,9 @@ onInit req = do
   riderPhoneCountryCode <- decRider.mobileCountryCode & fromMaybeM (PersonFieldNotPresent "mobileCountryCode")
   riderPhoneNumber <- decRider.mobileNumber & fromMaybeM (PersonFieldNotPresent "mobileNumber")
   bppBookingId <- booking.bppBookingId & fromMaybeM (BookingFieldNotPresent "bppBookingId")
+  city <- case booking.merchantOperatingCityId of
```

Comment:
Should be handled in FromTType as suggested.

Reply:
Done.